R/fun_makefile_create.R

```diff
@@ -59,27 +60,29 @@ deepstate_create_makefile <-function(package,fun_name){
   dir.create(file.path(fun_path, paste0(fun_name,"_output")), showWarnings=FALSE, recursive=TRUE)
   file.create(makefile_path, recursive=TRUE)
 
-  obj.file.list <- Sys.glob(file.path(package, "src/*.so"))
-  if(length(obj.file.list) <= 0){
-    # try to generate the shared object before failing
+  shared_objects <- file.path(package, "src/*.so")  
+  if (length(Sys.glob(shared_objects)) <= 0) {
+    # ensure that the debugging symbols are embedded in the shared object
     makevars_file <- file.path(package, "src", "Makevars")
     if (dir.exists(file.path(package, "src"))) {
-      makevars_content <- "PKG_CXXFLAGS += -g \n"
-      write(makevars_content, makevars_file, append=TRUE)
+        makevars_content <- "PKG_CXXFLAGS += -g \n"
```

Comment:
is there any reason to have a different/non-standard 4 space indent here? if not can you please change back to 2 space indent, which seems conssitent with the surrounding code.

Reply:
My mistake, I updated this portion of code with commit f957075